src/components/reports/views.tsx

```diff
@@ -0,0 +1,461 @@
+         </td>
+         <td className="govuk-table__cell">
+           {props.orgQuotaMapping[organization.relationships.quota.data.guid].entity.name}
+         </td>
+         <td className="govuk-table__cell">
+           {moment(organization.created_at).format(DATE)}
+         </td>
+         <td className="govuk-table__cell">
+           {new Date() > props.orgTrialExpirys[organization.guid] ? 'Expired' : 'Expires'}
+           {' '}
+           {moment(props.orgTrialExpirys[organization.guid]).fromNow()}
+         </td>
+       </tr>
+       ))}
+      </tbody>
+    </table>
+
+    <h2 className="govuk-heading-m">Billable accounts</h2>
+
+    <p className="govuk-body">
+      There {props.billableOrgs.length === 1 ? 'is' : 'are'}
+      {' '}
+      {props.billableOrgs.length}
+      {' '}
+      billable {props.billableOrgs.length === 1 ? 'organisation' : 'organisations'}.
+    </p>
+
+    <p className="govuk-body">
+      Sorted by age; newest first.
+    </p>
+
+    <table className="govuk-table">
+      <thead className="govuk-table__head">
+        <tr className="govuk-table__row">
+          <th className="govuk-table__header">Organisation</th>
+          <th className="govuk-table__header">Owner</th>
+          <th className="govuk-table__header">Quota</th>
+          <th className="govuk-table__header">Creation date</th>
+          <th className="govuk-table__header">Time since creation</th>
+        </tr>
+      </thead>
+      <tbody className="govuk-table__body">
+      {props.billableOrgs.map(organization => (
+        <tr key={organization.guid} className="govuk-table__row">
+          <td className="govuk-table__cell">
+            <a href={props.linkTo('admin.organizations.view', {organizationGUID: organization.guid})} className="govuk-link">
+              {organization.name}
+            </a>
+          </td>
+          <td className="govuk-table__cell">
+            {organization.metadata.annotations.owner === undefined ?
+              'Unknown' : organization.metadata.annotations.owner}
+          </td>
+          <td className="govuk-table__cell">
+            {props.orgQuotaMapping[organization.relationships.quota.data.guid].entity.name}
+          </td>
+          <td className="govuk-table__cell">
+            {moment(organization.created_at).format(DATE)}
+          </td>
+          <td className="govuk-table__cell">
+            Created {moment(organization.created_at).fromNow()}
+          </td>
+        </tr>
+      ))}
+      </tbody>
+    </table>
+  </>);
+}
+
+export function CostReport(props: ICostReportProperties): ReactElement {
+  return(<>
+    <h1 className="govuk-heading-l">
+      Billables for {props.date}
+    </h1>
+
+    <div className="govuk-grid-row">
+      <div className="govuk-grid-column-one-quarter">
+        <h2 className="govuk-heading-m">
+          {JSON.stringify(props.billableEventCount)}
```

Comment:
Using `JSON.stringify` on a number 🤔

Reply:
It's a nice artefact from the past `{{ billableEventCount | dump }}` :D

Changing.